ES/ESdd029.xml

```diff
@@ -290,21 +288,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                         </item>
                         <item xml:id="e1">
-                           <desc type="StampExlibris">Shelfmark C3-IV-277 (<locus target="#2r"/>)</desc></item>
+                           <locus target="#2r"/>
+                           <desc type="StampExlibris">Shelfmark C3-IV-277</desc></item>
                         
                         <item xml:id="e2">
-                           <desc type="StampExlibris">Shelfmark 278 (<locus target="#2r"/>)</desc></item>
+                           <locus target="#2r"/>
+                           <desc type="StampExlibris">Shelfmark 278</desc></item>
                         
                         <item xml:id="e3">
                            <locus target="#2r"/>
-                           <desc type="StampExlibris">The title of the manuscript <foreign xml:lang="gez">ዳዊት</foreign> and the internal number of the monastic library (278).</desc>
+                           <desc type="StampExlibris">The title of the manuscript <foreign xml:lang="gez">ዳዊት፡</foreign> and the internal number of the monastic library (278).</desc>
                         </item>
                           <item xml:id="e4">
-                             <desc type="Unclear">The midpoint of the <ref target="#ms_i1.1"/>‘Psalms of David’ (here in <ref type="work" corresp="LIT2000Mazmur#Ps78">Psalm 78</ref>) is marked with two asterisks on <locus target="#60r"/></desc>
+                             <locus target="#60r"/>
+                             <desc type="Unclear">The midpoint of the <ref target="#ms_i1.1"/>‘Psalms of David’ (here in <ref type="work" corresp="LIT2000Mazmur#Ps78">Psalm 78</ref>) is marked with two asterisks.</desc>
                           </item>
       
                         <item xml:id="e5">
-                           <locus target="#6r #6v #7v # 8r #18v #19r #34r  #36v #38v  #43r #52v #59r #61v #65v #67r #69r #72v #78r #80r #82 #83v #90v #92r #93v #104v #105r #111r #116v #123r #130r #131v #132r #133r #134v #135r #135v"/>
+                           <locus target="#6r #6v #7v #8r #18v #19r #34r #36v #38v  #43r #52v #59r #61v #65v #67r #69r #72v #78r #80r #82 #83v #90v #92r #93v #104v #105r #111r #116v #123r #130r #131v #132r #133r #134v #135r #135v"/>
```

Comment:
I still see two spaces between 36 and 38 - probably won't be printed but better avoid, as each space is normally seen as the beginning of a new element